R/github-labels.R

```diff
@@ -1,22 +1,41 @@
-#' Create standard github labels for labelling issues.
+#' Manage GitHub issue labels.
 #'
-#' This function creates new labels and changes colours (if needed).
-#' It does not generally remove labels, unless you explicit ask to
-#' remove GitHub's defaults.
+#' @description `use_github_labels()` creates new labels and/or changes label
+#' colours. It does not generally remove labels, unless you explicitly ask to
+#' remove the default GitHub labels that are not present in the labels you
+#' provide via `labels`.
 #'
-#' @param delete_default If `TRUE`, will remove default labels.
+#' `gh_labels()` returns the labels and colours commonly used by tidyverse
```

Comment:
Should this include `tidy` since it's a tidyverse convention?

Reply:
`gh_labels()` --> `tidy_labels()` in 7d7a544719593807769a86af06e4052ca2ff697b